lib/api/db/index.ts

```diff
@@ -11,7 +11,7 @@ export const getDB = () => {
   return db
 }
 
-export const setUpMongoDB = async () => {
+export const seedDatabase = async () => {
```

Comment:
I think that we should separate seeding methods for mongodb and other drivers

Reply:
the database abstraction is the same , it should be the same seeding mechanism